app/javascript/dashboard/components/widgets/DashboardApp/Frame.vue

```diff
@@ -38,6 +38,18 @@ export default {
       return this.currentChat?.meta?.sender?.id;
     },
   },
+
+  mounted() {
+    window.onmessage = e => {
+      if (
+        typeof e.data !== 'string' ||
+        e.data === 'chatwoot-dashboard-app:fetch-info'
```

Comment:
@pranavrajs I think this should be.
```suggestion
        e.data !== 'chatwoot-dashboard-app:fetch-info'
```

Reply:
That makes sense. Thanks for testing this. While debugging with Tooljet, I used `includes`, later moved it to `===`.